packages/preview/bamdone-aiaa/0.1.0/template/main.typ

```diff
@@ -0,0 +1,229 @@
+//***************************************************************
+// AIAA TYPST TEMPLATE
+// 
+// The author of this work hereby waives all claim of copyright
+// (economic and moral) in this work and immediately places it 
+// in the public domain; it may be used, distorted or 
+// in any manner whatsoever without further attribution or notice
+// to the creator. The author is not responsible for any liability 
+// from the usage or dissemination of this code.
+//
+// Author: Isaac Weintraub, Alexander Von Moll
+// Date: 06 NOV 2023
+// BAMDONE!
+//***************************************************************
+
+#import "@preview/bamdone-aiaa:0.1.0": aiaa
+
+#show: aiaa.with(
+    title: "Preparation of Papers for AIAA Technical Conferences",
+    bibliography-file: "refs.bib",
```

Comment:
This should use a `bibliography("...")` call, otherwise the file path will be resolved relative to the package directory, leading to a failure. You can take a look at the charged-ieee template to see how it's done there. The whole path handling will be improved in the future, but in the meantime, packages unfortunately have to work around it.

Reply:
I was able to follow in the example from `charged-ieee`. Thank you for the suggestion!